api/swagger.yml

```diff
@@ -1138,6 +1138,23 @@ components:
           description: The id of the created metarange
           type: string
 
+    UsageEvent:
+      type: object
+      properties:
+        class:
+          description: stats event class
+          type: string
+        name:
+          description: stats event name
+          type: string
```

Comment:
I am not sure what these two are.  Perhaps an example might clarify this documentation?